wordleCracker.py

```diff
@@ -164,9 +253,38 @@ def solve():
                 #narrowing the selection.
                 potWords = narrow(avWords(guess(myGuess)))
             except:
+                """
+                We shouldn't use an exception for an expected case. 
+                Also, it's always good practice to at least log out the exception. Why? Well there
+                will be one day, perhaps long from now, where your code breaks for a reason that
+                is new or not expected. And on that day, you will want to know what the exception 
+                was. Trust me.
+                ```

```

Comment:
Good idea - I often have to change try: & except: into an if True: to test for exceptions.